convert2rhel/unit_tests/actions/report_test.py

```diff
@@ -96,10 +103,16 @@
                     "message": "OVERRIDABLE MESSAGE",
                 },
                 "ErrorAction": {"status": STATUS_CODE["ERROR"], "error_id": "ERROR", "message": "ERROR MESSAGE"},
+                "TestAction": {
+                    "status": STATUS_CODE["ERROR"],
+                    "error_id": "DANGER",
+                    "message": "Test that two of the same status works",
+                },
             },
             False,
             [
                 "(ERROR) ErrorAction.ERROR: ERROR MESSAGE",
+                "(ERROR) TestAction.DANGER: Test that two of the same status works",
```

Comment:
I always associate `DANGE` as `WARNING` in a lot of contexts... Maybe it is really associated with an imminent `ERROR`?

Reply:
Yeah, I can see danger meaning either "warning" or "full stop" depending on the context. I will change the error_id.